knxdclient/encoding.py

```diff
@@ -273,5 +278,7 @@ def decode_value(value: EncodedData, t: KNXDPT) -> Any:
     elif t is KNXDPT.ENUM8:
         # The raw int val is returned. The User code must construct the correct Enum type if required.
         return val[0]
+    elif t is KNXDPT.COLOUR_RGB:
+        return f'{(val[2] + (val[1] << 8) + (val[0] << 16)):X}'
```

Comment:
`decode_value()` should return the same data type and format as `encode_value()` takes for the respective DPT. In addition, its easier to process a list/tuple of integers than parsing the hex string. 

Thus, we should return a `bytes` array of the three values here (or, better, as suggested above: a tuple of three ints).